micromind/networks/yolo.py

```diff
@@ -353,8 +353,11 @@ class Yolov8Neck(nn.Module):
         Depth multiple of the Darknet.
     """
 
-    def __init__(self, filters=[256, 512, 768], up=[2, 2], d=1):
+    def __init__(
+        self, filters=[256, 512, 768], up=[2, 2], heads=[True, True, True], d=1
```

Comment:
maybe here we can add/improve the docstrings to explain in detail the behaviour of each parameter